data/json/item_groups.json

```diff
@@ -9057,7 +9057,8 @@
       ["distaff_spindle", 50],
       ["carding_paddles", 50],
       ["knitting_needles", 50],
-      ["pointy_stick", 50]
+      ["pointy_stick", 50],
+	  ["atlatl", 20]
```

Comment:
Or thematically-related: put the new gun next to other models of the same caliber/skill (pistol with other pistols, etc).  The important thing to do is to **not** just dump it at the end of the list.